core/src/banking_stage.rs

```diff
@@ -118,10 +111,30 @@ pub struct BankingStageStats {
     transaction_processing_elapsed: AtomicU64,
 }
 
+#[derive(Debug, Default)]
+struct VoteSourceCounts {
+    receive_and_buffer_packets_count: AtomicUsize,
+    dropped_packets_count: AtomicUsize,
+    newly_buffered_packets_count: AtomicUsize,
+    newly_buffered_forwarded_packets_count: AtomicUsize,
```

Comment:
If I want to understand how many gossip votes are landing (vs. just getting discarded because they are dups with what we've gotten over TPU), can I do that w/ these metrics?

Reply:
don't think we've ever had the ability to do that. and the labels can actually be incorrect because of a bug: https://github.com/anza-xyz/agave/pull/5813#discussion_r2052971304

Reply:
We used to when each thread could actually execute/commit but lost it when we merged execution into jus the TPU thread.

It would be nice to get this visibility back (maybe dedup on receive or something?), but doesn't need to gate this PR